src/renderer/gl_engine/tvgGlGeometry.h

```diff
@@ -180,26 +180,25 @@ class GlPoint
 };
 
 class GlStageBuffer;
+class GlRenderTask;
 
 class GlGeometry
 {
 public:
-
+    GlGeometry() = default;
     ~GlGeometry();
 
-    bool tesselate(const RenderShape& rshape, RenderUpdateFlag flag, GlStageBuffer* gpuBuffer);
+    bool tesselate(const RenderShape& rshape, RenderUpdateFlag flag);
     void disableVertex(uint32_t location);
-    void draw(const uint32_t location, RenderUpdateFlag flag);
+    bool draw(GlRenderTask* task, GlStageBuffer* gpuBuffer, RenderUpdateFlag flag);
     void updateTransform(const RenderTransform* transform, float w, float h);
     float* getTransforMatrix();
 
 private:
-    uint32_t mFillVertexOffset;
-    uint32_t mFillIndexOffset;
-    uint32_t mFillCount;
-    uint32_t mStrokeVertexOffset;
-    uint32_t mStrokeIndexOffset;
-    uint32_t mStrokeCount;
+    Array<float> mFillVertex = {};
```

Comment:
For consistency, I'd like to suggest you to remove "m" from the member variables name.